pkgs/by-name/fi/filterpath/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+  testers,
+  filterpath
+}:
+stdenv.mkDerivation (finalAttrs: {
+  name = "filterpath";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "Sigmanificient";
+    repo = "filterpath";
+    rev = finalAttrs.version;
+    hash = "sha256-vagIImWQQRigMYW12lw+Eg37JJ2yO/V5jq4wD3q4yy8=";
+  };
+
+  makeFlags = [
+    "CC=${stdenv.cc}/bin/cc"
+    "PREFIX=${placeholder "out"}/bin"
+  ];
+
+  passthru.tests.version = testers.testVersion {
+    package = filterpath;
+  };
+
+  doCheck = true;
+  checkPhase = ''
+    echo "[`pwd`]" | ./filterpath | grep "`pwd`"
+  '';
+
+  meta = {
+    homepage = "https://github.com/Sigmanificient/filterpath";
+    description = "Retrieve a valid path from a messy piped line.";
+    license = lib.licenses.gpl3Only;
+    maintainers = with lib.maintainers; [ sigmanificient ];
+    mainProgram = "filterpath";
```

Comment:
`stdenv` doesn't set default platforms, so you have to set one by yourself, otherwise it will display misleading information on nixos-search.